### PR TITLE
add htmx webjars module

### PR DESCRIPTION
**Pull request by atomfrede**
Moving the webjars module to webjars-locator, adding a htmx-webjars module.

updates #4339

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
maybe adding a test in the CI ?
see https://github.com/jhipster/jhipster-lite/blob/main/tests-ci/generate.sh

**Comment by atomfrede** [comment]
Yes good point.

File: src/main/java/tech/jhipster/lite/generator/server/webjars/locator/application/WebjarsLocatorApplicationService.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package tech.jhipster.lite.generator.server.webjars.locator.application;
+
+import org.springframework.stereotype.Service;
+import tech.jhipster.lite.generator.server.webjars.locator.domain.WebjarsLocatorModuleFactory;
+import tech.jhipster.lite.module.domain.JHipsterModule;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+
+@Service
+public class WebjarsLocatorApplicationService {
```
Comment (murdos) [inline src/main/java/tech/jhipster/lite/generator/server/webjars/locator/application/WebjarsLocatorApplicationService.java:9]: FYI we can also design this with only one `Webjars` module with a factory that would expose multiple methods for htmx, webjars-locator, ... and multiple `JHipsterModuleResource` beans in a unique `WebjarsModuleConfiguration`.

Comment (atomfrede) [reply src/main/java/tech/jhipster/lite/generator/server/webjars/locator/application/WebjarsLocatorApplicationService.java:9]: Do we have an example for that? How will it be displayed in the ui?

Comment (murdos) [reply src/main/java/tech/jhipster/lite/generator/server/webjars/locator/application/WebjarsLocatorApplicationService.java:9]: Yes, see e.g. [src/main/java/tech/jhipster/lite/generator/server/springboot/cucumber/infrastructure/primary/CucumberModuleConfiguration.java](https://github.com/jhipster/jhipster-lite/blob/b57eb3e99c3e5189c160fd0b541412a999d08578/src/main/java/tech/jhipster/lite/generator/server/springboot/cucumber/infrastructure/primary/CucumberModuleConfiguration.java#L14)

The UI display is generated from the JHipsterModuleOrganization data of the JHipsterModuleResource, it's not related to the origin (aka the package) of the JHipsterModuleResource. Module resources with the same feature are regrouped, and a module resource is displayed according to its dependencies.

Comment (atomfrede) [reply src/main/java/tech/jhipster/lite/generator/server/webjars/locator/application/WebjarsLocatorApplicationService.java:9]: Thanks,  I think its a good idea, such that more webjars can be added easily if needed. Will change it accordingly

**Review by murdos — DISMISSED** [review]

**Pull request merged**